Public/Src/Engine/Processes/SandboxedProcess.cs

```diff
@@ -197,10 +196,59 @@ public async Task<SandboxedProcessResult> GetResultAsync()
                     return null;
                 }
 
-                return detouredProcess.GetJobObject()?.GetPeakMemoryUsage();
+                ulong? currentPeakWorkingSet = null;
+                ulong? currentPeakPagefileUsage = null;
+
+                var jobObject = detouredProcess.GetJobObject();
+                if (jobObject == null || !jobObject.TryGetProcessIds(out uint[] childProcessIds) || childProcessIds.Length == 0)
+                {
+                    return null;
+                }
+
+                foreach (uint processId in childProcessIds)

```

Comment:
>foreach (uint processId in childProcessIds) [](start = 16, length = 43)

Would this affect perf.?

Reply:
it seems negligible in my test builds. I already measure how much time we spend as a part of QueryRamUsage counter.